cmd/generate.js

```diff
@@ -0,0 +1,29 @@
+const commander = require('commander');
+const chalk = require('chalk');
+const { generateRandomString } = require('../app/utils');
+
+const generate = new commander.Command('generate');
+
+const SECRET_TYPES = {
+  JWT_SECRET: 'JWT_SECRET',
+};
+
+generate.description('generate secrets to use in the config file');
```

Comment:
This description is duplicate. You should use a more general description.

Reply:
Do you have a better idea? Something like `generate things`? But that doesnt sound good.

Reply:
maybe ``generation functions for strings needed for config files``